### ISSUE TITLE
Exception in thread "main" java.lang.NoClassDefFoundError: javax/activation/

### ISSUE THREAD

**Issue by vastoncosta** [transpect/docx2tex#46]
Hello,

After updating jvm, I can no longer use docx2tex to produce tex files from docx files with the ./d2t command. I use mac OS via terminal for conversions and currently I have the following version of java:
java version "1.8.0_211"
Java (TM) SE Runtime Environment (build 1.8.0_211-b12)
Java HotSpot (TM) 64-Bit Server VM (build 25.211-b12, mixed mode).
I would like to know if the problem is really with the java version of my computer, if someone else has already encountered this problem and, if possible, what solution should I take to remedy the problem.

Thank you very much in advance.

Follow the log generated.

2           Exception in thread "main" java.lang.NoClassDefFoundError: javax/activation/    DataSource
  3         at java.base/java.lang.Class.getDeclaredMethods0(Native Method)
  4         at java.base/java.lang.Class.privateGetDeclaredMethods(Class.java:31    72)
  5         at java.base/java.lang.Class.getMethodsRecursive(Class.java:3313)
  6         at java.base/java.lang.Class.getMethod0(Class.java:3299)
  7         at java.base/java.lang.Class.getMethod(Class.java:2112)
  8         at com.xmlcalabash.core.XProcRuntime.initializeSteps(XProcRuntime.ja    va:317)
  9         at com.xmlcalabash.core.XProcRuntime.<init>(XProcRuntime.java:272)
 10         at com.xmlcalabash.drivers.Main.run(Main.java:100)
 11         at com.xmlcalabash.drivers.Main.main(Main.java:83)
 12 Caused by: java.lang.ClassNotFoundException: javax.activation.DataSource
 13         at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Builti    nClassLoader.java:583)
 14         at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadCla    ss(ClassLoaders.java:178)
 15         at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
 16         ... 9 more

**Comment by mkraetke** [comment]
Hi @vastoncosta ,
thank you for reporting this issue. Can you give me your OSX version? I mainly work with Linux and Windows, but I'll try to reproduce this error on a Mac.
Best, Martin

**Comment by vastoncosta** [comment]
Hi, @mkraetke .

Sorry for the delay!!

My OSX version is Mojave version 10.14.5 (18F132).

Thanks again.

Best, Vaston

**Comment by mkraetke** [comment]
Hi @vastoncosta,

Unfortunately, I was not able to reproduce your error, but the `DataSource` class should be part of the JRE, so the bug should be related to your JRE installation. I tested docx2tex successfully on a MB Pro with MacOS v10.13.6. Here is what I did:

1. I got the JRE v1.8.0_211-b12 from the Oracle website and installed it
2. I opened a terminal and created `~/.bash_profile`
3. Then I edited `~/.bash_profile` and set `$JAVA_HOME`: 
`export JAVA_HOME=/Library/Internet\ Plug-Ins/JavaAppletPlugin.plugin/Contents/Home`
4. Finally, I run docx2tex and I got no error.

**Comment by carlomontec** [comment]
Hello,
I have the very same error with the current java Ubuntu 18.04:
```
± |master ?:3 ✗| → java --version
openjdk 11.0.4 2019-07-16
OpenJDK Runtime Environment (build 11.0.4+11-post-Ubuntu-1ubuntu218.04.3)
OpenJDK 64-Bit Server VM (build 11.0.4+11-post-Ubuntu-1ubuntu218.04.3, mixed mode, sharing)
```
do I need to install also the JRE? I am very ignorant about Java, my apologies.
 Regards

**Comment by mkraetke** [comment]
Fixed with 5ae4085619a385898ea81bb225f9735db390f0f6
